### DISCUSSION TITLE
AI Content Identification

### DISCUSSION THREAD

**Discussion by johnwdaniels** [directus/directus#16212] [Feature Requests]
It would be great if we could create a flow or feature that scanned data and files and was able to generate tags. If a user has hundreds or thousands of files and records, it would be a long process to look at a picture and manually add tags.

This could possibly be done using other fields in records to generate tags perhaps, or scanning with an AI engine to auto detect pieces of a picture for example and tag a file that way.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
